src/VisualStudio/Core/Test/Utilities/VsNavInfoHelpers.vb

```diff
@@ -26,23 +26,23 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Utilities.VsNavInfo
                    End Sub
         End Function
 
-        Public Function Package(expectedName As String) As NodeVerifier
+        Public Function PackageNode(expectedName As String) As NodeVerifier
```

Comment:
Renamed to avoid conflicts with `Package` type coming from newer VS platform packages.